pkg/reconciler/kubernetes/tektonresult/transform.go

```diff
@@ -675,3 +679,90 @@ func updateEnvWithDBSecretName(props v1alpha1.ResultsAPIProperties) mf.Transform
 		return nil
 	}
 }
+
+// injectPostgresUpgradeSupport modifies the postgres container to support automatic
+// upgrade from PostgreSQL 13 to PostgreSQL 15. The wrapper script checks the PG_VERSION
+// file in the data directory and sets POSTGRESQL_UPGRADE=copy if an upgrade is needed,
+// triggering the sclorg container's built-in upgrade mechanism.
+func injectPostgresUpgradeSupport() mf.Transformer {
```

Comment:
I see this 
```
// Note: PostgreSQL upgrade transformer is NOT needed for Kubernetes
```
But function is defined in Kubernetes do we still need this ??

Reply:
No. This is a leftover and I removed it. Initially the update was implemented for both k8s and OCP, but then during testing I found in k8s we use images from bitname which do not have those v13, v15 etc versions, so the issue does not affect them.